src/Service/CheckRecaptchaV2.php

```diff
@@ -34,6 +34,7 @@ public function check($challenge, $response)
             curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
             curl_setopt($curl, CURLOPT_TIMEOUT, app('config')->get('recaptcha.options.curl_timeout', 1));
             curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, false);
+            curl_setopt($curl, CURLOPT_PROXY, app('config')->get('recaptcha.options.curl_proxy', NULL));
```

Comment:
Actually, can you modify this so it only updates `CURLOPT_PROXY` if it's added in the config? If the value is being set elsewhere (e.g. in php.ini), this will clobber that value.